setup.py

```diff
@@ -1,5 +1,10 @@
 from setuptools import find_packages, setup
 
+# To install extra package from local by using direct references. 
+# In production, this should be installed from pypi
+import os
+path_to_my_project = f"{os.getcwd()}/plugins/tabular_plugin"
```

Comment:
This will cause the app to crash if there is no tabular plugin to install. I know this is temporal, but it can give errors, maybe check if directory exists first